### ISSUE TITLE
Fixed format SUBST converts with extra colon (:)

### ISSUE THREAD

**Issue by m-tyler** [BrianGarland/vscode-rpgfree#37]
I am at version  0.0.21 and I am seeing this conversion problem.

If I have this code
```     C                   IF        @Y > 0
     C                   SUBST(P)  @AMTA:@Y      @AMTA@
     C                   ELSE
     C                   CLEAR                   @AMTA@
     C                   ENDIF
```
it ends up converting to this
```         IF @Y > 0;
            @AMTA@ = %Subst(@AMTA:@Y:);
          ELSE;
            CLEAR   @AMTA@;
          ENDIF;
```
Notice the line with %SUBST() has a trailing colon with no variable or constant.

Thanks, Matt

**Comment by BrianGarland** [comment]
Will be fixed in 0.0.23

**Comment by m-tyler** [comment]
I confirm this fix works for me.
